src/components/rhelGraphCard/rhelGraphCard.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { withBreakpoints } from 'react-breakpoints';
```

Comment:
future: 
something to consider, import syntax consistency around `react-breakpoints` looks like we used the default import on `pageLayout`

Reply:
ah yes, good call!

Reply:
i'm actually seeing `ReactBreakpoints` as a default export in the docs, and `withBreakpoints` as a named import. Is there another export?

Reply:
nope, you got it!